cves/CVE-2013-0861.yml

```
@@ -127,7 +138,12 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    From the top of my head, I remember atleast 10 developers that worked on 
+    utlis.c between the first VCC and fix. I noticed that many of the commits 
```

Typo of `utils.c` as `utlis.c`
```suggestion
    utils.c between the first VCC and fix. I noticed that many of the commits 
```